config/pipelines/pacbio.yml

```diff
@@ -560,6 +560,100 @@ default: &default
                 - :well
                 - :sample
               with: :row_type
+        column_order:
```

Comment:
I suppose we could also add order as an attribute for each field but not sure which is best?